spec/classes/debian_spec.rb

```diff
@@ -134,7 +135,23 @@
               ).with_content(
                 # This section varies for different releases
                 /\Unattended-Upgrade::Origins-Pattern\ {\n
-                \t"origin=Debian,archive=stable,label=Debian-Security";\n
+                \t"origin=Debian,codename=\${distro_codename},label=Debian-Security";\n
+                };/x
+              )
+            end
+          end
+        when 'buster'
+          context 'with defaults on Debian 10 Buster' do
+            it do
+              is_expected.to create_file(file_unattended).with(
+                owner: 'root',
+                group: 'root',
+                mode: '0644'
+              ).with_content(
+                # This section varies for different releases
+                /\Unattended-Upgrade::Origins-Pattern\ {\n
```

Comment:
why is `U` `\`ed?

Reply:
Hi @igalic, I don't know that construct by hand to be honest, I copied it from other places in the same file.

For me it looks like a multiline regex definition. Maybe they are needed to mask possible regex characters like `-` and `:` in the whole phrase `\Unattended-Upgrade::Origins-Pattern\` ?

Reply:
since we already use this elsewhere, it seems fair to accept this.